pkg/cmd/cli/backup/create.go

```diff
@@ -132,7 +132,7 @@ func (o *CreateOptions) BindFlags(flags *pflag.FlagSet) {
 	f = flags.VarPF(&o.IncludeClusterResources, "include-cluster-resources", "", "Include cluster-scoped resources in the backup")
 	f.NoOptDefVal = "true"
 
-	f = flags.VarPF(&o.DefaultVolumesToRestic, "default-volumes-to-restic", "", "Use restic by default to backup all pod volumes")
+	f = flags.VarPF(&o.DefaultVolumesToFsBackup, "default-volumes-to-fs-backup", "", "Use pod volume file system backup by default for volumes")
```

Comment:
Have we decided to introduce the breaking change in the CLI? Please don't forget to add release note for this if we have decided

Reply:
Yes, this has been decided because we don't want users to use the `default-volumes-to-restic `for Kopia path.
We will mention this change in the coming upgrade guide doc, the new pod volume backup doc as well as the release note.